src/mod/applications/mod_callcenter/mod_callcenter.c

```diff
@@ -2133,45 +2277,28 @@ static void *SWITCH_THREAD_FUNC outbound_agent_thread_run(switch_thread_t *threa
 			/* Busy: Do Not Disturb, Circuit congestion */
 			case SWITCH_CAUSE_NORMAL_CIRCUIT_CONGESTION:
 			case SWITCH_CAUSE_USER_BUSY:
-				delay_next_agent_call = (h->busy_delay_time > delay_next_agent_call? h->busy_delay_time : delay_next_agent_call);
+				delay_next_agent_call = (h->busy_delay_time > delay_next_agent_call ? h->busy_delay_time : delay_next_agent_call);
+				cc_agent_noanswer_count_status_update(h->agent_name, h->member_uuid, h->member_session_uuid, cause, h->agent_busy_status, 
+										h->busy_count, h->max_busy);
```

Comment:
I don't like it that we send individually the busy_count and max_busy here...
I'm not sure if we need the busy_count max_busy( and all the other one) any where else.... If not, then I would move in that function a SELECT that check for these values.  This would also make a better behavior, where if you ring during 60 seconds an agent, if you change the agent value during that 60 seconds, the changes won't be applied until the next call.

Reply:
here i got lost.. if you could purpose your change, i'm happy to test it locally. 

note: here i had forget to set the not_registered_delay_time and remove the hardcode 5seconds.

Reply:
I changed that in the pull request I've made.  I've removed reference to the different count and max that exist, and loaded them directly in that function from the agent.